pkgs/development/tools/continuous-integration/buildbot/plugins.nix

```diff
@@ -176,4 +176,26 @@
     };
   };
 
+  gitea = buildPythonPackage rec {
+    pname = "buildbot-gitea";
+    version = "1.7.2";
+
+    src = fetchPypi {
+      inherit pname version;
+      hash = "sha256-zfHq7xmvKKVl+OuEXvsQg2T23gJGbGl3rKeTkc/oFG0=";
+    };
+
+    buildInputs = [ buildbot-pkg ];
+    propgagatedBuildInputs = [ requests ];
+    checkInputs = [ buildbot requests ];
```

Comment:
Shouldn't this be nativeCheckInputs? Also it shouldn't be required to duplicate entries with propgagatedBuildInputs